src/partitioner/local/MetisPartitioner.cpp

```diff
@@ -1009,3 +1009,125 @@ void MetisPartitioner::writeSerializedCompositeMasterFiles(std::string part) {
     masterFileMutex.unlock();
     partitioner_logger.log("Serializing done for central part " + part, "info");
 }
+
+void MetisPartitioner::setPathAndGraphID(int graphID) {
```

Comment:
This function is not useful. I do not see any point of having such function.

Reply:
This method is no longer there in the updated hdfs implementation.